BUILDIT.md

```diff
@@ -5469,13 +5469,13 @@ to the `LiveView` server during the mounting phase.
 
 Open `assets/js/app.js`
 and locate the 
-`let liveSocket = new LiveSocket()` variable.
+`let live_socket = new live_socket()` variable.
```

Comment:
does this _work_? 🤷‍♂️ 
Thought `LiveSocket()` was defined by:
https://github.com/dwyl/mvp/blob/49f40422923cf7ee7bc53367d1e4287d2dd519eb/assets/js/app.js#L5
Looks like you haven't updated the invocation in `app.js` just assigned it to a different variable name. 🔍

Reply:
This is in the `BUILDIT.md`, forgot to change it back in the documentation after fixing it on the `app.js` file.

Fixed.